ghcide/src/Development/IDE/GHC/Compat/Core.hs

```diff
@@ -0,0 +1,964 @@
+{-# LANGUAGE CPP               #-}
```

Comment:
While very convenient to have every Identifier listed, might be a maintenance nightmare.
Rather re-export whole modules? Or split up into more smaller modules?

Reply:
Reexport all

Reply:
Sorry, do you mean to re-export modules, or like this, every identifier?

Reply:
Reexporting all modules seems like a bad idea, since we are omitting certain definitions on purpose. My compromise would be to re-export simple modules, such as 'NameEnv', 'Predicate' and the like, but keep the explicit listing for more complicated functions and types, such as 'DynFlags'.